includes/admin/class-wc-rest-woopay-session-controller.php

```diff
@@ -52,15 +53,19 @@ public function register_routes() {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_session_data( WP_REST_Request $request ): WP_REST_Response {
+		$cart_token = $this->validate_cart_token( $request->get_header( 'cart_token' ) );
+		$payload    = JsonWebToken::get_parts( $cart_token )->payload;
```

Comment:
To avoid a potentially fatal error, we may want to ensure the `JsonWebToken` class exists before we use it.

[Here's an example](https://github.com/Automattic/woocommerce-payments/blob/a1fcf877642d953232122058e1a72beae4168464/includes/woopay/class-woopay-session.php#L83-L84) of how we've done it in the past.